tests/testlib.py

```diff
@@ -750,4 +753,36 @@ def setup_gssproxy(testdir, logfile, env):
         print("Attach and start debugging, then press enter to continue.")
         input()
 
+    gssproxy_reload(testdir, gproc.pid, {
+        'signal': 0 , 'key': 'Initialization complete' })
+
     return gproc, socket
+
+def gssproxy_reload(testdir, pid, action=None):
+
+    if action is None:
+        action = dict()
```

Comment:
Would it be easier here to just add two parameters - `signal` and `key` - rather than a dict?

Reply:
this was future proofing
I thin it is fine